.github/workflows/workflow.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       shell: bash
       run: |
         npm run build
-        if [ "$(git status --porcelain | wc -l)" -gt "0" ]; then
+        if [ "$(git diff --ignore-space-at-eol | wc -l)" -gt "0" ]; then
```

Comment:
`tslib.js` in `@azure/core-auth` uses CRLF line endings which gets flagged by this step.  Ignoring EOL characters.